lenovo/thinkpad/p16s/amd/gen1/default.nix

```diff
@@ -0,0 +1,6 @@
+{ config, lib, pkgs, ... }:
+
+{
+  # For mainline support of rtw89 wireless networking
+  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "5.16") pkgs.linuxPackages_latest;
```

Comment:
Please also reference this file in flake.nix and the README.md.

Reply:
sure!